src/firebaseAuthService.js

```diff
@@ -0,0 +1,12 @@
+(function() {
+  "use strict";
+
+  function FirebaseAuthService($firebaseAuth, firebaseRef) {
+    return $firebaseAuth(firebaseRef);
+  }
+  FirebaseAuthService.$inject = ['$firebaseAuth', 'firebaseRef'];
```

Comment:
This is the proposed structure in the Angular Styleguide, which is supported by the core team.

I would like for us to move that way eventually. I think starting here is fine because of how small and isolated it is from the rest of the codebase.